web/docs/use/transform/README.md

```diff
@@ -2,8 +2,36 @@
 
 VAST has powerful features for transforming [data in
 motion](#modify-data-in-motion) and [data at rest](#modify-data-at-rest). Both
-aspects rely on [pipelines](/docs/understand/query-language/pipelines) as
-building block.
+aspects rely on [pipelines](/docs/understand/language/pipelines) as building
+block.
+
+## Define a static pipeline
```

Comment:
I removed "static.

The newly introduced terminology is actually *open pipeline*, so I think `vast.pipelines` remains valid.

Reply:
Those are different things to me. Any pipeline definition can be used as a logical operator, independent of whether they're open or closed. Whether they're closed or opened only changes the context in which they can be instantiated.

Reply:
Right now, we don't have `vast.operators` but only `vast.pipelines` in the config file. So this would probably at most be a change in a future version.

But you basically just said that using "pipelines" is not wrong:

> Any pipeline definition can be used as a logical operator

So then what's wrong with `vast.pipelines`?

Reply:
In a quick call we agreed to slightly rephrase this: We want to call the building blocks operators, so users define operators here in their config (and as such we should rename this to `vast.operators` sometime in the future), because then the term pipeline unambiguously refers to something instantiated.

Reply:
Right. I'm leaving this for now as is, because we should do one global rename from `vast.pipelines` to `vast.operators`.